test/API/H5_api_test.h

```diff
@@ -62,11 +59,12 @@ extern const char *test_path_prefix;
 
 #define UNUSED(o) (void)(o);
 
-#define H5_API_TEST_FILENAME_MAX_LENGTH 1024
-
 /* The maximum size of a dimension in an HDF5 dataspace as allowed
  * for this testing suite so as not to try to create too large
  * of a dataspace/datatype. */
 #define MAX_DIM_SIZE 16
 
-#endif
+// TODO: Remove
+#define MT_API_TEST_FUNC_OUTER(func) func
```

Comment:
Looks like this can be removed now. I also saw a few references where it was still being used.